Bottom: 834cffa9709212668b8b51540ddd6924f3a4517d
Top:    290a928f8c72ccc9c3c524f544c25d7ed2d2d320
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-02-19 15:29:52 +0000

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d43ab45..e32b64b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1326,7 +1326,7 @@ static int
 clearPad_callback_solid (const BoxType * b, void *cl)
 {
   PadTypePtr pad = (PadTypePtr) b;
-  if (!XOR (TEST_FLAG (ONSOLDERFLAG, pad), SWAP_IDENT) && pad->mask)
+  if (!XOR (TEST_FLAG (ONSOLDERFLAG, pad), SWAP_IDENT) && pad->Mask)
     ClearPadSolid (pad, true);
   return 1;
 }
